arches/install/arches-templates/project_name/webpack/webpack.common.js

```diff
@@ -198,6 +201,7 @@ module.exports = () => {
                                                     '\x1b[33m%s\x1b[0m',  // yellow
                                                     `"${templatePath}" has failed to load. Retrying (${failureCount} / 5)...`
                                                 );
+                                                await new Promise(r => setTimeout(r, 8000));
```

Comment:
Can you remove this change in favor of changing it on a project-by-project basis? So far most installations do not have an issue with the speed of these requests ( we give webpack 5 chances to fetch the correct template )

Reply:
**I have removed this.**

Webpack failing after 5 retries is a consistent issue on both my Windows and Linux tests but both environments are on the lower tier of VM available and so it fails to handle all the requests and repeat requests done in such a short time (Windows seems worse at handling all the requests than Linux but they both will fail all 5 times for a few requests per run). This is mostly an issue with multiple languages enable because of the extra network activity from the 302 redirects for each request.

I will simply make this change locally for my installations.